docs/usage-and-billing.mdx

```diff
@@ -0,0 +1,158 @@
+tags:
+  - Semgrep Cloud Platform
+  - Semgrep OSS
+  - Team & Enterprise Tier
+hide_title: true
+---
+
+import MoreHelp from "/src/components/MoreHelp";
+
+<ul id="tag__badge-list">
+  {Object.entries(frontMatter)
+    .filter((frontmatter) => frontmatter[0] === "tags")[0]
+    .pop()
+    .map((value) => (
+      <li class="tag__badge-item">{value}</li>
+    ))}
+</ul>
+
+<!-- # Pricing and billing -->
+
+# Usage and billing 
+
+Learn about usage computation and other aspects of your Semgrep licenses.
+
+:::note
+- This document is for **Semgrep Pro** users; Semgrep OSS Engine does not have any usage limit. Compare [<i class="fa-regular fa-file-lines"></i>  Semgrep Pro and Semgrep OSS](/semgrep-pro-vs-oss).
+:::
+
+## Usage 
+
+All Semgrep Pro products are free under the **Team** tier for **10 monthly contributors**. These products include:
+
+- Semgrep Cloud Platform
+- Semgrep Code
+- Semgrep Supply Chain
+- Semgrep Secrets (beta)
+
+A **contributor** is someone who has made at least **one** commit to a Semgrep-scanned private repository within the last month, starting from the **date of license purchase** if a license was purchased, or the date of account creation, for accounts using Semgrep within usage limits.
+
+Any type of Semgrep Pro scan counts towards the contributor total. This includes:
+
+- Scanning with any Semgrep Pro product (Code, Supply Chain, and so on.)
+- Full scans on a repository or partial scans on a pull or merge request.
+
+### Usage computation
+
+Contributors are calculated using `git log` over the **past 30 days** (a rolling interval), not the beginning of each month. The start date is either:
+
+- The date of your license purchase.
+- The date of account creation, if you and your team are within usage limits.
+
+**Bots** and similar automations are excluded from this count.
+
+### Exceeding the free usage limit
+
+Semgrep scans stop when the usage limit is exceeded. You can resume scanning through the following:
+
+* A one-time 30-day free trial that starts automatically when the usage limit is exceeded for the first time.
+* By purchasing additional licenses.
+* By waiting for the next billing cycle.
+
+If the very first scan exceeds the usage limit, Semgrep still completes the first scan and a one-time 30-day free trial automatically starts. After the trial concludes, if scans are run on private repositories that exceed the usage limit, scans will not run until additional licenses are purchased.
+
+### Usage limit on public projects
+
+Public projects have no limits on contributors.
+
+### Semgrep CLI commands subject to usage limits
+
+The `semgrep scan` command is subject to the usage limit only if the scan is by a logged-in or authenticated contributor. Semgrep computes contributor counts for any logged-in scan command (for example, `semgrep ci`, `semgrep scan`, and so on) when the Pro Engine, Supply Chain, or Pro rules are used.
```

Comment:
```suggestion
The `semgrep scan` command is subject to the usage limit only if the scan is by a logged-in contributor. Semgrep computes contributor counts for any logged-in scan command (for example, `semgrep ci`, `semgrep scan`, and so on) when the Pro Engine, Supply Chain, or Pro rules are used.
```
Not sure what the extra "or authenticated" means here. If it has a distinct meaning, happy to brainstorm ideas for a different word :)